types/chromecast-caf-receiver/cast.framework.messages.d.ts

```diff
@@ -629,9 +629,12 @@ export class Track {
     trackId: number;
 
     /**
-     * The type of track.
-     */
-    type: TrackType;
```

Comment:
This property still exists and it's documented, you shouldn't delete it.

Reply:
thanks for pointing out, have added back.